Dockerfile

```diff
@@ -18,19 +19,33 @@ RUN yum install -y curl which patch make git \
 # Switch to directory with sources
```

Comment:
move this under section where the user is created... so that you don't have to `chown`

Reply:
done